packages/framework/src/Model/Administrator/Administrator.php

```diff
@@ -61,7 +61,8 @@ class Administrator implements UserInterface, Serializable, UniqueLoginInterface
     protected $email;
 
     /**
-     * @var \Doctrine\Common\Collections\ArrayCollection|\Shopsys\FrameworkBundle\Model\Administrator\AdministratorGridLimit[]
+     * @var \Shopsys\FrameworkBundle\Model\Administrator\AdministratorGridLimit[]|\Doctrine\Common\Collections\ArrayCollection
```

Comment:
```suggestion
     * @var \Shopsys\FrameworkBundle\Model\Administrator\AdministratorGridLimit[]|\Doctrine\Common\Collections\Collection
```

Initially the `ArrayCollection` is true, but once the Doctrine hydrates the entity, it hydrates the `$gridLimit` as a `PersistentCollection`, and these two classes are different. The only think that they have in common is the interface `Collection`

Reply:
Please change it in all occurences in this PR